bin/setup

```diff
@@ -18,6 +18,7 @@ chdir APP_ROOT do
   system! "gem install bundler --conservative"
   system! "bundle config --local cache_all true"
   system! "bundle config --local cache_all_platforms true"
+  system! "bundle config --local ignore_messages.rpush true"
```

Comment:
Thanks for this! 

```
Updating files in vendor/cache
Fetching gem metadata from https://rubygems.org/......
  * rpush-5.3.0.gem
  * rpush-redis-1.1.0.gem
Bundle complete! 155 Gemfile dependencies, 352 gems now installed.
Bundled gems are installed into `./vendor/cache`
```

works as advertised - gem installs, no post-install message.